react-exercise/src/components/creator-card/CreatorCard.scss

```diff
@@ -1,14 +1,13 @@
 $number-of-posts: 3;
 $creator-card-width: 342px;
 $creator-profile-size: 80px;
-$post-preview-width: $creator-card-width / $number-of-posts;
-$profile-offset-from-post: ($post-preview-width - $creator-profile-size) / 2; /* Center the profile image */
+$post-preview-width: calc($creator-card-width / $number-of-posts);
```

Comment:
I wasn't sure if it is the appropriate way or if I should maybe use math.div or something else